### PR TITLE
Adapted OpenMote-CC2538 sensor drivers to Contiki sensor API.

### PR DESCRIPTION
**Pull request by peretuset**
This PR adapts the OpenMote-CC2538 sensor drivers to the Contiki sensor API.

### CODE REVIEW

**Comment by g-oikonomou** [comment]
Thank you Pere. @alignan Antonio, I had a (very quick) look at this and it looks sane. Any thoughts?

**Comment by alignan** [comment]
Happy to merge! :+1:

File: platform/openmote-cc2538/dev/adxl346.c [hunk 1]
```diff
@@ -185,23 +186,86 @@ adxl346_is_present(void)
...
+
+  result = (1000 * accel) / 256;
+
+  return (int16_t) result;
+}
+/*---------------------------------------------------------------------------*/
+static void
+adxl346_calibrate_offset(void)
+{
+	int32_t accum_x = 0;
```
Comment (alignan) [inline platform/openmote-cc2538/dev/adxl346.c]: Keep the spaces consistent

File: platform/openmote-cc2538/dev/adxl346.c [hunk 2]
```diff
@@ -185,23 +186,86 @@ adxl346_is_present(void)
...
+  config[1] = 0;
+  i2c_burst_send(ADXL346_ADDRESS, config, sizeof(config));
+  config[0] = ADXL346_OFSY_ADDR;
+  config[1] = 0;
+  i2c_burst_send(ADXL346_ADDRESS, config, sizeof(config));
+  config[0] = ADXL346_OFSZ_ADDR;
+  config[1] = 0;
+  i2c_burst_send(ADXL346_ADDRESS, config, sizeof(config));
+
+	uint16_t i;
```
Comment (alignan) [inline platform/openmote-cc2538/dev/adxl346.c]: Here as well

File: platform/openmote-cc2538/dev/adxl346.c [hunk 3]
```diff
@@ -218,17 +282,27 @@ status(int type)
...
+  int16_t accel;
   if(!enabled) {
     PRINTF("ADXL346: sensor not started\n");
     return ADXL346_ERROR;
   }
 
   if(type == ADXL346_READ_X) {
     return adxl346_read_accel(ADXL346_DATAX0_ADDR, ADXL346_DATAX1_ADDR);
-  } else if(type == ADXL346_READ_Y) {
+      } else if(type == ADXL346_READ_Y) {
```
Comment (alignan) [inline platform/openmote-cc2538/dev/adxl346.c]: Mind the extra spaces

File: platform/openmote-cc2538/dev/adxl346.c [hunk 4]
```diff
@@ -251,6 +326,11 @@ configure(int type, int value)
     }
   }
 
+  if(type == ADXL346_CALIB_OFFSET && enabled) {
+  	adxl346_calibrate_offset();
```
Comment (alignan) [inline platform/openmote-cc2538/dev/adxl346.c]: Extra spaces there

**Comment by g-oikonomou** [comment]
This looks pretty reasonable. @peretuset can you please put files through uncrustify? The ADXL driver at least needs a bit of love.

**Comment by peretuset** [comment]
Done it! Let me know if there is anything else that needs love... :)

**Comment by peretuset** [comment]
Thanks Antonio!

**Pull request merged**